sdk/storage/storage-file-datalake/samples/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@azure/identity": "^1.1.0-preview1",
+    "@azure/identity": "1.1.0-preview1",
```

Comment:
The samples get published to docs.microsoft.com
I don't believe we want such samples to use preview versions of dependencies

cc @willmtemple, @KarishmaGhiya

Reply:
cc @jonathandturner